app/Policies/TransactionPolicy.php

```diff
@@ -25,10 +25,9 @@ public function delete(User $user, Transaction $transaction): bool
         }
 
         if ($transaction->checkout->name == Checkout::studentsCouncil()->name) {
-            return $user->hasRole(Role::STUDENT_COUNCIL, Role::ECONOMIC_LEADER)
-                || (
-                    $user->hasRole(Role::STUDENT_COUNCIL, Role::ECONOMIC_MEMBER)
-                    && $transaction->receiver_id == $user->id
+            return $user->hasRole(Role::STUDENT_COUNCIL, Role::ECONOMIC_VICE_PRESIDENT)
+                || (//is this okay this way?
```

Comment:
We needed the check like this to ensure that economic members could only delete the transaction they received. You can just leave this as a economic_vice_president check. (Are we sure that the president will not need any help collecting the transactions?)

Reply:
In recent years anyone can pay KKT by bank transfer, so help shouldn't be needed.